src/test/java/org/schemaspy/testcontainer/OracleIT.java

```diff
@@ -0,0 +1,133 @@
+package org.schemaspy.testcontainer;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.schemaspy.Config;
+import org.schemaspy.cli.CommandLineArguments;
+import org.schemaspy.model.Database;
+import org.schemaspy.model.ProgressListener;
+import org.schemaspy.model.Table;
+import org.schemaspy.model.TableColumn;
+import org.schemaspy.service.DatabaseService;
+import org.schemaspy.service.SqlService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.testcontainers.containers.OracleContainer;
+import org.testcontainers.jdbc.ext.ScriptUtils;
+import org.testcontainers.utility.JdbcDriverUtil;
+
+import javax.script.ScriptException;
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.SQLException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class OracleIT {
+
+    @Autowired
+    private SqlService sqlService;
+
+    @Autowired
+    private DatabaseService databaseService;
+
+    @Mock
+    private ProgressListener progressListener;
+
+    @MockBean
+    private CommandLineArguments arguments;
+
+    @MockBean
+    private CommandLineRunner commandLineRunner;
+
+    private static final String driverPath = "ext-lib/ojdbc*";
+
+    private static Database database;
+
+    @ClassRule
+    public static OracleContainer oracleContainer = new OracleContainer<>()
+            .assumeDocker()
+            .withDrivers(driverPath);
```

Comment:
Are you sure you've pulled the latest changes. I only ask since travis-ci, marked the test as skipped and not failed.